portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoColumnLocalServiceImpl.java

```diff
@@ -540,7 +540,7 @@ else if (type == ExpandoColumnConstants.STRING_LOCALIZED) {
 			Map<Locale, String> defaultValuesMap =
 				(Map<Locale, String>)defaultData;
 
-			Locale defaultLocale = LocaleUtil.getDefault();
+			Locale defaultLocale = LocaleUtil.getSiteDefault();
 
```

Comment:
@crodriguezy, I think the change is not right. Since ExpandoColumn doesn't belong to site level and it belongs to company level. We can't add ExpandoColumn in any sites.